pkg/gatherers/clusterconfig/gather_logs.go

```diff
@@ -129,6 +130,10 @@ func filterLogs(
 
 	for scanner.Scan() {
 		line := scanner.Text()
+		if messagesToSearch == nil {
+			result += line + "\n"
+		}
```

Comment:
This opens the new functionality for this function. It would be nice to unify it with https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/clusterconfig/operators_pods_and_events.go#L254 
Just saying..... otherwise LGTM, but we need to check with @jholecek-rh about potential impact on the rules.

Reply:
Yeah, I started refactoring to have only one place to gather logs, but then decided it doesn't worth the efforts right now.